RocketChat/Rocket.Chat
# [FIX] Go command duplicating subfolder path on iframes.

## Proposed changes (including videos or screenshots)





## Issue(s)

closes #22737
## Steps to test or reproduce

- Run Rocket.Chat in a subfolder (the following steps apply to running `ROOT_URL=http://localhost:3000/test meteor`)
- Create a .html file with the following code 
```
<!DOCTYPE html>
<html lang="en" style="width: 100%;height: 100%;">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Document</title>
</head>
<body style="width: 100%;height: 100%;">
    <iframe style="width: 100%;height: 100%;" src="http://localhost:3000/test"></iframe>
</body>
</html>
```
- in the rocket.chat instance, go to admin > general and disable "Restrict Access inside any iframe"
- still in general, go to Iframe Integration and enable both "Send" and "Receive"
- Now open the html file in your browser.
- Open the console, and set your context to be the iframe, like this
![image](https://user-images.githubusercontent.com/40830821/127388211-853f78ec-676b-4471-a7e2-593a624f5787.png)

- Still in the console, run: `window.postMessage({externalCommand: "go", path: "/admin"}, "*")`
- Should redirect to admin. To check the url, just type in the console: `window.location` and hit enter. Should display a variable like this:
![image](https://user-images.githubusercontent.com/40830821/127388461-3c3a1d88-6e5e-466b-b43d-486f0c574b12.png)

PS: Every time you refresh the page, the console context has to be set to the ifram again
## Further comments
